dask_expr/tests/test_merge.py

```diff
@@ -76,23 +76,29 @@ def test_merge_indexed(how, pass_name, sort, shuffle_backend):
 
 
 @pytest.mark.parametrize("how", ["left", "right", "inner", "outer"])
-def test_broadcast_merge(how):
+@pytest.mark.parametrize("npartitions", [None, 22])
+def test_broadcast_merge(how, npartitions):
     # Make simple left & right dfs
-    pdf1 = lib.DataFrame({"x": range(20), "y": range(20)})
-    df1 = from_pandas(pdf1, 4)
-    pdf2 = lib.DataFrame({"x": range(0, 20, 2), "z": range(10)})
-    df2 = from_pandas(pdf2, 1)
+    pdf1 = lib.DataFrame({"x": range(40), "y": range(40)})
+    df1 = from_pandas(pdf1, 20)
+    pdf2 = lib.DataFrame({"x": range(0, 40, 2), "z": range(20)})
+    df2 = from_pandas(pdf2, 2)
 
-    df3 = df1.merge(df2, on="x", how=how)
+    df3 = df1.merge(
+        df2, on="x", how=how, npartitions=npartitions, shuffle_backend="tasks"
```

Comment:
Does the `shuffle_backend` matter here?

Reply:
we don't do broadcast merges for disk (don't know why)